README.md

```diff
@@ -1,16 +1,16 @@
-# Odin project sign up form
+# Odin dashboard project
 
-> One paragraph statement about the project.
+> The project contains some information .
```

Comment:
This description is not descriptive enough, you need to explain what the project is, very brief not too long but at the same time should explain about the project